src/Frontend/src/components/failedmessages/EditRetryDialog.vue

```diff
@@ -106,7 +106,7 @@ function isContentTypeSupported(contentType: string) {
 
 function getMessageIntent() {
   const intent = findHeadersByKey("NServiceBus.MessageIntent");
-  return intent.value;
+  return intent?.value ?? "";
```

Comment:
This looks weird, @PhilBastian; why this change?
I looked more closely at this component, and it is totally untyped, which is scary!
I also looked at `getContentType()`, which is quite similar to this function.
So, let's add types to this component to be on the safe side.

Reply:
@PhilBastian I refactor this and added types, can you review those changes